setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import find_packages, setup
+
+from version import get_version
+
+setup(
+    name="digital-land-data-manager",
+    version=get_version(),
+    author="MHCLG Digital Land Team",
+    author_email="DigitalLand@communities.gov.uk",
+    license="MIT",
+    url="https://github.com/digital-land/pipeline",
```

Comment:
This should probably be the URL of this repo rather than an old link to `digital-land-python` repo.